command/agent/command.go

```diff
@@ -88,6 +100,25 @@ func (c *Command) readConfig() *Config {
 		return nil
 	}
 
+	// Split the servers.
+	if servers != "" {
+		cmdConfig.Client.Servers = strings.Split(servers, ",")
+	}
+
+	// Parse the meta flags.
+	if len(meta) != 0 {
+		cmdConfig.Client.Meta = make(map[string]string)
```

Comment:
You can specify the map length here since we already know it to save a few allocations.